sdk/java-sdk-protobuf/src/main/scala/kalix/javasdk/impl/action/ActionsImpl.scala

```diff
@@ -199,7 +200,10 @@ private[javasdk] final class ActionsImpl(
   override def handleUnary(in: ActionCommand): Future[ActionResponse] =
     services.get(in.serviceName) match {
       case Some(service) =>
-        val span = telemetries(service.serviceName).buildSpan(service, in)
+        // This future is always completed before this method is called because that future completes right after the proxy discovery
+        // which always happens before any message can be processed by any component
```

Comment:
No future here anymore (and same comment in the other files)

Reply:
removed